logic.js

```diff
@@ -0,0 +1,187 @@
+stored_posts =
+[
+    {
+        id: 1,
+        text: "First post!"
+    },
+    {
+        id: 2,
+        text: "Second post!"
+    }
+];
+
+stored_comments =
+[
+    {
+        id: 1,
+        text: " haha1",
+        linked_post: 1
+    },
+    {
+        id: 2,
+        text: " haha2",
+        linked_post: 1
+    },
+    {
+        id: 3,
+        text: " haha3",
+        linked_post: 1
+    },
+    {
+        id: 4,
+        text: " baba1",
+        linked_post: 2
+    },
+    {
+        id: 5,
+        text: " baba2",
+        linked_post: 2
+    },
+    {
+        id: 6,
+        text: " baba3",
+        linked_post: 2
+    },
+];
+
+const draw_comment = function(ul, comment)
+{
+    let li = document.createElement('li');
```

Comment:
You should try to use jQuery. It is modern and clean and a lot of jobs will use it.

Reply:
thanks Amelia, when I finished coding this tweeter: we learned about jQuery (cuz I did all this job in 1 day) but I started using jQuery after :) thanks